src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -794,12 +790,6 @@ - (void)startCallWithWriteable:(id<GRXWriteable>)writeable {
 
     [self sendHeaders];
     [self invokeCall];
-
-    // Connectivity monitor is not required for CFStream
-    char *enableCFStream = getenv(kCFStreamVarName);
-    if (enableCFStream != nil && enableCFStream[0] != '1') {
-      [GRPCConnectivityMonitor registerObserver:self selector:@selector(connectivityChanged:)];
```

Comment:
`connectivityChanged` seems to be unused now. Can it be removed? Same comemnt for `connectivityChange` in GRPCChannelPool.m.